planet/src/main/webapp/js/authentication.js

```diff
@@ -74,3 +74,67 @@ function signIn() {
         // ...
     });
 }
+
+function signInWithGoogle() {
+    const provider = new firebase.auth.GoogleAuthProvider();
```

Comment:
Maybe try renaming these consts to be more specific, const should always be in capital case ex: GOOGLE_PROVIDER

Reply:
Fixed!